less12.py

```diff
@@ -0,0 +1,52 @@
+class Moving:
+    def move(self):
+        raise NotImplemented
+
+
+class Animal(Moving):
+    def voice(self):
+        raise NotImplemented
+
+
+class Transport(Moving):
+    def launch(self):
+        raise NotImplemented
+
+
+class Duck(Animal):
+    def voice(self):
+        return 'quack'
+
+    def move(self):
+        return 'swim'
+
+
+class Tiger(Animal):
+    def voice(self):
+        return 'roar'
+
+    def move(self):
+        return 'runs'
+
+
+class Car(Transport):
+    def __init__(self, status='not_started'):
```

Comment:
we cant launch our car without using `launch` method.
Remove you param `status` from `__init__`

Reply:
Only param, but leave it like attribute with value `no_started`